docs/reference/getting-started.asciidoc

```diff
@@ -158,6 +158,8 @@ On macOS, Elasticsearch can also be installed via https://brew.sh[Homebrew]:
 brew install elasticsearch
 --------------------------------------------------
 
+If installation succeeds, Homebrew will finish by saying that you can start Elasticsearch by entering `elasticsearch`. Do that now. The expected response is described below, under `Successfully running node`.
```

Comment:
Maybe you could add an anchor like `[[successfully-running-node]]` to the section in question and then reference it with an internal cross references `<<....>>`? I think that would make navigating to the next step even easier.

Reply:
Thank you, @cbuescher. I lost track of your suggestion. Now I have implemented it, after a long delay.